news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -229,6 +231,38 @@ protected function parseArticle($objArticle, $blnAddArchive=false, $strClass='',
 			$responseTagger->addTags(array('contao.db.tl_news.' . $objArticle->id));
 		}
 
+		// schema.org information
+		$objTemplate->getBasicSchemaOrgData = static function () use ($objTemplate, $objArticle): array
```

Comment:
Should we move this above the `parseArticles` hook so people can replace the function if they want?

Reply:
I don't think they should. You can modify the JSON-LD in the central `JsonLdEvent` and modify whatever you want which is exactly why I have designed the JsonLdManager to collect and only then display it :)